lighthouse-core/audits/is-on-https.js

```diff
@@ -27,6 +27,7 @@ class HTTPS extends Audit {
       category: 'Security',
       name: 'is-on-https',
       description: 'Site is on HTTPS',
+      helpText: 'All sites should be protected with HTTPS, even ones that don\'t handle sensitive data. HTTPS prevents intruders from tampering with or passively listening in on the communications between your app and your users. <a href="https://developers.google.com/web/tools/lighthouse/audits/https" target="lighthouse" rel="noreferrer noopener">Learn more</a>.',
```

Comment:
I think we could add a nod to things like Service Worker requiring HTTPS given it's referenced elsewhere in the report.